editor/src/components/canvas/commands/reparent-element-command.spec.tsx

```diff
@@ -83,12 +78,7 @@ describe('runReparentElement', () => {
     ])
     const originalEditorState = renderResult.getEditorState().editor
 
-    const reparentCommand = reparentElement(
-      'always',
-      targetPath,
-      childInsertionPath(newParentPath),
-      'use-deprecated-insertJSXElementChild',
-    )
+    const reparentCommand = reparentElement('always', targetPath, childInsertionPath(newParentPath))
```

Comment:
Is it sure that this is always a childInsertionPath?

Reply:
no, because PASTE_JSX_ELEMENT has a big chunk of code that deals with wrapping elements into fragments on paste

Reply:
these more poignant tests should address that https://github.com/concrete-utopia/utopia/pull/3638/commits/31a96a0a3ecce576c5f9a5b5f732e8e01f282abb (cc @bkrmendy )